src/access.ts

```diff
@@ -0,0 +1,11 @@
+/** @module access */
+
+// eslint-disable-next-line import/prefer-default-export
+export enum access {
```

Comment:
There's a TS convention where type names are virtually always upper cased, and b/c enums are often used as types I think we should do the same here:

```suggestion
export enum Access {
```